main.go

```diff
@@ -451,8 +452,10 @@ func commitChanges(updatedList PackageList) error {
 	}
 
 	wt.Add(indexFile)
-
 	commitMessage := "Charts CI\n```"
+	if iconOverride {
```

Comment:
@adamkpickering 
the `autoUpdate -> generateChanges` functions are modifying the `index.yaml` with some other modifications at some cases.

the `overrideIcons` function is overriding the `index.yaml`, so it will generate new modification on top of the existing ones again. 

I would like to have 2 commits separately for debugging reasons in case something goes wrong. 
This way we can compare things through commits later.